### PR TITLE
Feature/angular es6

### PR DESCRIPTION
**Pull request by MichaelCastiau**

### CODE REVIEW

File: generators/nodejs/nodejs.js [hunk 1]
```diff
@@ -48,12 +47,20 @@ function getOptions() {
...
   debug.logger(`Clone backend version: ${tag}`);
   await gitclone(repository, tag);
-  debug.logger(`Copy files from repo`);
+  debug.logger('Copy files from repo');
+  if (config.frontend) {
+    debug.logger('Enable Frontend in Dockerfile.');
+    await replace({
+      files: './tmp/Dockerfile',
+      from: ['# RUN npm install', '# RUN npm run build'],
+      to: ['RUN npm install', 'RUN npm run  build'],
```
Comment (TriangleJuice) [inline generators/nodejs/nodejs.js]:
```suggestion
      to: ['RUN npm install', 'RUN npm run build'],
```

File: .editorconfig [hunk 2]
```diff
@@ -11,4 +11,4 @@ insert_final_newline = true
 
 [files/**]
 indent_style = space
-indent_size = 2
+indent_size = 2
```
Comment (TriangleJuice) [inline .editorconfig]: `insert_final_newline` not working for you?

File: .eslintrc [hunk 3]
```diff
@@ -34,4 +34,4 @@
     "nock": false,
     "request": false
   }
-}
+}
```
Comment (TriangleJuice) [inline .eslintrc]: `insert_final_newline` not working for you?

File: generators/angular/angular.js [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    this.generator = templateGenerator;
+  }
+
+  async start() {
+    updateLog('Preparing...');
+    await this.prepareDirectory();
+    await this.installAngular(this.configuration);
+    await this.installACPaasUI(this.configuration);
+    await this.createStarterTemplate(this.configuration);
+    updateLog('Done with front-end setup', 'cyan');
```
Comment (TriangleJuice) [inline generators/angular/angular.js]:
```suggestion
    updateLog('Done with frontend setup', 'cyan');
```
Kind suggestion to be consistent in naming things.

File: generators/angular/files/src/app/app.component.html.template.hbs [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      {{#if routing.add}}
+      <a routerLink="/home" class="a-button o-header__button">Home</a>
+      <a routerLink="/about" class="a-button o-header__button">About</a>
+      {{/if}}
+      {{#if auth}}
+      <aui-user-menu (login$)="login()" (logout$)="logout()"></aui-user-menu>
+      {{/if}}
+    </div>
+  </div>
+</header>
```
Comment (TriangleJuice) [inline generators/angular/files/src/app/app.component.html.template.hbs]: Maybe you could use the `aui-header` component for this?

File: generators/angular/files/src/app/app.component.html.template.hbs [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    <router-outlet></router-outlet>
+    {{else}}
+    <home-page></home-page>
+    {{/if}}
+  </div>
+</main>
+<aui-footer>
+  <div auiFooterBottom>
+    <aui-subfooter>
+      <aui-copyright domain="Stad Antwerpen"></aui-copyright>
```
Comment (TriangleJuice) [inline generators/angular/files/src/app/app.component.html.template.hbs]:
```suggestion
      <aui-copyright domain="stad Antwerpen"></aui-copyright>
```
We use lowercase `s` these days

File: generators/angular/files/src/app/app.component.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+	  max-width: 100%;
+	}
+	
+	main {
+	  flex: 1 0 auto;
+	  width: 100%;
+	
+	  @include to($screen-xs-max) {
+	    padding-left: 0;
+	  }
```
Comment (TriangleJuice) [inline generators/angular/files/src/app/app.component.scss]: What is this for?

File: generators/angular/files/src/app/app.module.ts.template.hbs [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    FooterModule,
+    HeaderModule,
+    LogoModule,
+    {{#if routing.add}}
+    AppRoutingModule,
+    {{/if}}
+    {{#if auth}}
+    UserMenuModule,
+    ServiceModule,
+    AvatarModule,
```
Comment (TriangleJuice) [inline generators/angular/files/src/app/app.module.ts.template.hbs]: We could discuss here whether to put these components in their own `aui.modules.ts` file, like we do in our other projects.

File: generators/angular/files/src/index.html.template.hbs [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+  <meta charset="utf-8">
+  <title>{{ name }}</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="apple-touch-icon" sizes="180x180" href="https://cdn.antwerpen.be/core_branding_favicons/citizens/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="https://cdn.antwerpen.be/core_branding_favicons/citizens/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="https://cdn.antwerpen.be/core_branding_favicons/citizens/favicon-16x16.png">
+  <link rel="manifest" href="https://cdn.antwerpen.be/core_branding_favicons/citizens/site.webmanifest">
+  <link rel="mask-icon" href="https://cdn.antwerpen.be/core_branding_favicons/citizens/safari-pinned-tab.svg" color="{{#if coreBranding}}#347ea6{{else}}#cf0039{{/if}}">
```
Comment (TriangleJuice) [inline generators/angular/files/src/index.html.template.hbs]: These should be variable too.

File: generators/angular/files/src/styles.scss.template.hbs [hunk 10]
```diff
@@ -0,0 +1,36 @@
+/* You can add global styles to this file, and also import other style files */
+@import "~@a-ui/core/src/styles/quarks";
+{{{branding.scss}}}
+
+body {
+  display: flex;
+  min-height: 100vh;
+  height: 200px; /*IE 11 flexbox min-height fix*/
```
Comment (TriangleJuice) [inline generators/angular/files/src/styles.scss.template.hbs]: This fix isn't necessary anymore I believe.

File: generators/angular/files/src/styles.scss.template.hbs [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+  color: $white;
+}
+
+.o-header__logo {
+  position: static;
+
+  img {
+    height: $spacer * 2;
+  }
+}
```
Comment (TriangleJuice) [inline generators/angular/files/src/styles.scss.template.hbs]: I believe these styles are fixed with the latest core branding.

File: generators/angular/files/src/app/app.component.html.template.hbs (hunk 5 above)
Comment (MichaelCastiau) [reply generators/angular/files/src/app/app.component.html.template.hbs]: I was wondering if it exists, but I can't seem to find it ? https://digipolisantwerp.github.io/acpaas-ui_angular/modules/analytics

File: generators/angular/files/src/styles.scss.template.hbs (hunk 10 above)
Comment (MichaelCastiau) [reply generators/angular/files/src/styles.scss.template.hbs]: This is fixed in Edge, but still a bug in IE 11

File: generators/angular/files/src/app/app.component.scss (hunk 7 above)
Comment (MichaelCastiau) [reply generators/angular/files/src/app/app.component.scss]: It was included by previous developer, but not needed anymore, so removed it

File: generators/angular/files/src/index.html.template.hbs (hunk 9 above)
Comment (MichaelCastiau) [reply generators/angular/files/src/index.html.template.hbs]: What exactly should be variable?

**Pull request merged**